connectors/src/start_worker.ts

```diff
@@ -27,6 +28,27 @@ import logger from "./logger/logger";
 
 setupGlobalErrorHandler(logger);
 
+const pinoAdapter: Logger = {
+  log: (level: LogLevel, msg: string, meta: object) =>
+    ({
+      TRACE: logger.trace,
+      DEBUG: logger.debug,
+      INFO: logger.info,
+      WARN: logger.warn,
+      ERROR: logger.error,
+    })[level](meta ?? {}, msg),
+  info: (msg: string, meta: object) => logger.info(meta ?? {}, msg),
+  warn: (msg: string, meta: object) => logger.warn(meta ?? {}, msg),
+  error: (msg: string, meta: object) => logger.error(meta ?? {}, msg),
+  debug: (msg: string, meta: object) => logger.debug(meta ?? {}, msg),
+  trace: (msg: string, meta: object) => logger.trace(meta ?? {}, msg),
+};
+
+// Install once per process — before creating Worker/Client
+Runtime.install({
+  logger: pinoAdapter,
+});
```

Comment:
Can't we use directly the logger in the install? Curious about the reason we need to redeclare the logging methods

Reply:
I was more thinking about something like that:
```
Runtime.install({
  logger,
});
```
Because it seemed to me that the `logger` comes from pino, and the `pinoAdapter` seems to just pipe the same method.

But actually the `Logger` type comes from temporal and not pino, and yeah I just saw after that the params were inverted.